src/WebUI/ImageDetails/ImageDetails.tsx

```diff
@@ -261,14 +266,15 @@ export class ImageDetails extends BaseComponent<IImageDetailsProperties, IImageD
 
     private static _renderLayersSizeCell(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<IImageLayer>, imageLayer: IImageLayer): JSX.Element {
         // Currently size data is not present in imageLayer
-        const textToRender = "-";
+        const textToRender = imageLayer.size || "-";
         const itemToRender = defaultColumnRenderer(textToRender);
         return renderTableCell(rowIndex, columnIndex, tableColumn, itemToRender);
     }
 
     private static _renderLayersAgeCell(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<IImageLayer>, imageLayer: IImageLayer): JSX.Element {
         // Currently created data is not present in imageLayer
-        const itemToRender = <Ago date={new Date()} format={AgoFormat.Extended} />;
+        const layerCreatedOn = imageLayer.createdOn ? new Date(imageLayer.createdOn) : new Date();
+        const itemToRender = <Ago date={layerCreatedOn} format={AgoFormat.Extended} />;
```

Comment:
check with PM on extented or normal format

Reply:
Changed to Compact format as per spec